front-end/src/app/reports/f3/create-workflow/create-f3-step1.component.ts

```diff
@@ -123,6 +123,10 @@ export class CreateF3Step1Component extends FormComponent implements OnInit {
   });
 
   public thisYear = new Date().getFullYear();
+  readonly defaultReportTypeCategory =
```

Comment:
Does this ticket include updating Form 3 default coverage date logic as well? If so, I'm thinking the ticket should be updated to reflect this (especially for QA purposes).